### DISCUSSION TITLE
Navigateur Brave et Récompenses Brave

### DISCUSSION THREAD

**Discussion by linuxmrcmd** [AmauriC/tarteaucitron.js#1149] [General]
Bonjour,

Avec ce navigateur et l'activation de Récompenses Brave, le bandeau 'Accepter, Refuser ...' apparait furtivement lors de l'affichage de la page.

Est il possible d'éviter cela?
A+
Linuxmr

**Comment by AmauriC** [comment]
Bonjour :)

Pas sûr de comprendre, le bandeau s'affiche et disparaît ?
Si oui c'est un souci/fonctionnalité côté Brave.

**Reply by linuxmrcmd** [reply]
C'est possible
Je viens de rééssayer en initilisant le bloqueur, et j'ai bien le bandeau

**Reply by linuxmrcmd** [reply]
C'est assez incompréhensible, un autre site, et le bandeau apparait et disparait

L'icone Affichée, qui devrait etre en bas à gauche, n'apparait pas

**Reply by linuxmrcmd** [reply]
Je confirme que Recompense Brave a un fonctionnement curieux, maintenant cela fonctionne sans rien faire
